packages/neos-ui/src/Containers/Modals/SelectNodeType/nodeTypeGroupPanel.js

```diff
@@ -40,10 +47,21 @@ class NodeTypeGroupPanel extends PureComponent {
         const {
             group,
             collapsedGroups,
-            onSelect
+            onSelect,
+            filterSearchTerm,
+            i18nRegistry
         } = this.props;
         const {name, label, nodeTypes} = group;
 
+        const filteredNodeTypes = (nodeTypes || [])
+            .filter(nodeType => {
+                const label = i18nRegistry.translate(nodeType.label, nodeType.label);
+                if (label.toLowerCase().search(filterSearchTerm.toLowerCase()) !== -1) {
```

Comment:
Don't we need to escapre regexp here? There's method for this in lodash i think.

Reply:
i'll check

Reply:
Should be fixed now!